shoes-core/spec/shoes/common/visibility_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+
+describe Shoes::Common::Visibility do
+  class VisibilityTester
+    include Shoes::Common::Visibility
+    include Shoes::Common::Style
+
+    attr_reader :gui
+
+    def initialize(gui)
+      @style = {}
+      @gui = gui
+    end
+  end
+
+  subject { VisibilityTester.new(gui) }
+
+  let(:gui) { double("gui", update_visibility: nil) }
+
+  it "hides" do
+    subject.hide
+    expect(subject.style[:hidden]).to eq(true)
+    expect(subject.gui).to have_received(:update_visibility)
+  end
+
+  it "shows" do
+    subject.show
+    expect(subject.style[:hidden]).to eq(false)
+    expect(subject.gui).to have_received(:update_visibility)
```

Comment:
I was like "but where do we `update_visibility` now?" - but there's a spec that it happens (great speccing @jasonrclark :heart: ) so I'm confident that we do it, so I'm good :d